### PR TITLE
Feature/188 app navigation

### PR DESCRIPTION
**Pull request by chris-lawton**

### CODE REVIEW

File: opentech/static_src/src/sass/apply/components/_nav.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    &__link {
+        @extend %h5;
+        position: relative;
+        display: inline-block;
+        padding: 20px 10px;
+        color: $color--white;
+        transition: color $transition;
+
+        &:hover {
```
Comment (martycoote) [inline opentech/static_src/src/sass/apply/components/_nav.scss]: &:focus ?

File: opentech/static_src/src/sass/apply/components/_link.scss [hunk 2]
```diff
@@ -47,4 +47,26 @@
...
+        @include media-query(tablet-portrait) {
+            display: none;
+        }
+    }
+
+    &--button-transparent {
+        @include button(transparent, $color--darkest-blue);
+        color: $color--white;
+
+        &:hover {
```
Comment (martycoote) [inline opentech/static_src/src/sass/apply/components/_link.scss]: &:focus ?

File: opentech/static_src/src/sass/apply/components/_nav.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                height: 5px;
+                background-color: $color--dark-blue;
+                content: '';
+                opacity: 0;
+                transition: opacity 0.5s cubic-bezier(0.2, 1.4, 0.67, 1.13);
+            }
+        }
+
+        &--active,
+        &:hover {
```
Comment (martycoote) [inline opentech/static_src/src/sass/apply/components/_nav.scss]: &:focus ?

**Comment by chris-lawton** [comment]
@todd-dembrey - for the active state to appear, you just need to add a `nav__link--active` to `<a class="nav__link">`

**Pull request merged**